modules/ROOT/nav.adoc

```diff
@@ -19,14 +20,15 @@
  ** xref:access-management-security.adoc[Configure Security in Anypoint Platform PCE]
  ** xref:access-management-disclaimer.adoc[Configure a Disclaimer in Anypoint Platform PCE]
 * xref:operating-about.adoc[Manage Anypoint Platform PCE]
- ** xref:operating-about.adoc[Close port 61009]
  ** xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore in Anypoint Platform PCE]
  ** xref:managing-via-the-ops-center.adoc[Ops Center]
```

Comment:
In this page, change the word "Server" for "Node" in step 2 of the "Add Additional Servers" section